posts/oversubscribed-tracks.md

```diff
@@ -0,0 +1,13 @@
+One of the biggest challenges we face is when tracks get too popular for our mentors to keep up. The wait time for students increases, and mentors have the very unsatisfying experience of seeing queues keep growing despite putting a huge amount of work in to try and keep on top.
+
+Solving this is far from trivial, but we believe we have some good solutions that will make things better (see our [recent blog post](https://exercism.io/blog/whats-next-for-exercism-aug-2019) for details). However, these will all take time to implement, and there is a very real problem today.
+
+We've therefore decided to add an "Oversubscribed" mode to tracks, where when the median wait time for core exercises becomes too long, we:
+- Restrict all new students joining a track to use Practice Mode
+- Disallow any new submissions in Practice Mode
```

Comment:
```suggestion
- Disallow any new submissions in Mentored Mode
```

Otherwise this contradicts the sentences above and below it

Reply:
Hmmm. No, this is correct. We don't allow any new submissions in Practice Mode. How can I make the sentence above less confusing?

Reply:
I'm not sure, but here is how I read it. Maybe that helps:

My understanding is that, when a track is oversubscribed, students will be forced to use Practice/Independant Mode instead of Mentored Mode so that mentors have a chance of catching up. But then this sentence states you don't allow any new submissions in Practice/Independant Mode. So if they aren't allowed to join Mentored Mode but can't submit anything in Practice Mode, where do they submit their solved exercises?

Or does this refer to submissions in the sense that they request mentoring?

Reply:
> Or does this refer to submissions in the sense that they request mentoring?

This. Yes. I'll clarify :)

Reply:
```suggestion
- Stop students in Practice Mode from requesting mentoring on their solutions (they can continue to use Practice Mode independently)
```